### ISSUE TITLE
Can't get ansel to work on my mac

### ISSUE THREAD

**Issue by cgarrigues** [infused/ansel#1]
```
cgarrigues@LM-AUN-00874682> irb --version                                                                                                             
irb 0.9.6(09/06/30)
cgarrigues@LM-AUN-00874682> irb                                                                                                                       
irb(main):001:0> require 'ansel'
=> true
irb(main):002:0> converter = ANSEL::Converter.new
=> #<ANSEL::Converter:0x007f98bb231620 @to_charset="UTF-8">
irb(main):003:0> converter.convert("\xB9\x004.59") # => "\U+FFC2\U+FFA34.59"
NameError: uninitialized constant ANSEL::Converter::StringScanner
    from /Library/Ruby/Gems/2.0.0/gems/ansel-2.0.0/lib/ansel/converter.rb:17:in `convert'
    from (irb):3
    from /usr/bin/irb:12:in `<main>'
irb(main):004:0>
```

**Comment by infused** [comment]
This should be fixed in version 2.0.1 which I just released a few minutes ago.

**Comment by cgarrigues** [comment]
Thank you, sir.,

On March 13, 2016 at 2:17:38 PM, Keith Morrison (notifications@github.commailto:notifications@github.com) wrote:

This should be fixed in version 2.0.1 which I just released a few minutes ago.
